src/tmx_json.c

```diff
@@ -205,39 +207,164 @@ static int pjson_tile(json_t *tile_el, tmx_tile **tile_headaddr) {
 	return 1;
 }
 
+
+/* This is a rather slow solution - anyone with more spare time, feel free to rework the whole structure of
+   the tiles list to make this faster */

```

Comment:
I'm not sure if I can create another pull request before the previous one was accepted (I've tried that before and it didn't seem to work), but I'll just comment here once I've done it. From what it looks like, Github just adds all new commits to a pull request as long as that pull request hasn't been accepted. In any case, we will hopefully find out today.